test/e2e/framework/test_context.go

```diff
@@ -200,12 +212,20 @@ func ViperizeFlags() {
 	RegisterClusterFlags()
 	flag.Parse()
 
-	viperFlagSetter := func(f *flag.Flag) {
-		if viper.IsSet(f.Name) {
-			glog.V(4).Infof("[viper config] Overwriting, found a settting for %v %v", f.Name, f.Value)
-			f.Value.Set(viper.GetString(f.Name))
-		}
-	}
-	// Each flag that we've declared can be set via viper.
-	flag.VisitAll(viperFlagSetter)
+	viper.Unmarshal(&TestContext)
+
+	/** This can be used to overwrite a flag value.
```

Comment:
i think so - we havent yet decided what the future of old flags is.  up to u though..